packages/beacon-node/src/network/core/networkCoreWorker.ts

```diff
@@ -85,11 +85,12 @@ new AsyncIterableBridgeHandler(getReqRespBridgeReqEvents(reqRespBridgeEventBus),
 );
 const reqRespBridgeRespCaller = new AsyncIterableBridgeCaller(getReqRespBridgeRespEvents(reqRespBridgeEventBus));
 
+let networkCoreWorkerMetrics: NetworkCoreWorkerMetrics | null = null;
 // respBridgeCaller metrics
 if (metricsRegister) {
-  const networkCoreWorkerMetrics = getNetworkCoreWorkerMetrics(metricsRegister);
+  networkCoreWorkerMetrics = getNetworkCoreWorkerMetrics(metricsRegister);
```

Comment:
```ts
const networkCoreWorkerMetrics = metricsRegister ? getNetworkCoreWorkerMetrics(metricsRegister) : null;
if (networkCoreWorkerMetrics) {
  //
}
```
has the benefit of not compiling if someone deletes the if statement

Reply:
very good idea!!  🚀

Reply:
I will make sure to do it like this in the future as well